components/cards/Profile.tsx

```diff
@@ -31,7 +33,7 @@ export default function Profile({
         <style>
           {`.slideUp{animation-duration:1s;animation-name:slideUp}.slideDown{animation-duration:1s;animation-name:slideDown}.flipInX{animation-duration:1s;animation-name:flipInX}@keyframes slideUp{from{opacity:0;transform:translateY(20px)}25%{opacity:0}to{opacity:1;transform:translateY(0)}}@keyframes slideDown{from{opacity:0;transform:translateY(-20px)}25%{opacity:0}to{opacity:1;transform:translateY(0)}}@keyframes flipInX{from{transform:perspective(400px) rotate3d(1,0,0,90deg);animation-timing-function:ease-in;opacity:0}40%{transform:perspective(400px) rotate3d(1,0,0,-20deg);animation-timing-function:ease-in}60%{transform:perspective(400px) rotate3d(1,0,0,10deg);opacity:1}80%{transform:perspective(400px) rotate3d(1,0,0,-5deg)}to{transform:perspective(400px)}}`}
         </style>
-        <div xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
This was done on purpose, it's a trick to render HTML inside an SVG in a GitHub `.md` file.
Based on: https://github.com/sindresorhus/css-in-readme-like-wat

Reply:
Typescript was not happy with that so I changed div tag to svg instead

Reply:
I see!
I did a test and it seems that it's working fine inside a GitHub `README.md` file, you can keep it!